core/src/subgraph/instance.rs

```diff
@@ -78,8 +78,8 @@ where
 
             // Create services for static offchain data sources
             if let DataSource::Offchain(ds) = &ds {
-                if let Some(source) = &ds.source {
-                    offchain_monitor.monitor(source.clone());
+                if ds.source.is_some() {
```

Comment:
Why this `is_some`, shouldn't we let it error if the ds is somehow missing a source?